### DISCUSSION TITLE
v2rayN 的vmess一个端口只能用一天，第二天就连不上，请问什么原因，怎么办？

### DISCUSSION THREAD

**Discussion by clmomoz** [v2fly/v2ray-core#3156] [Q&A]
不知为何一个端口只能用一天，一到第二天V2rayn就连不上，显示真连接延迟-1，网速timed out，同时下面显示错误：[Warning] [2611877094] app/proxyman/outbound: failed to process outbound traffic > proxy/vmess/outbound: failed to find an available destination > common/retry: [transport/internet/websocket: failed to dial WebSocket > transport/internet/websocket: failed to dial to (ws://IP地址/1469f7da):  > dial tcp IP 地址: i/o timeout transport/internet/websocket: failed to dial WebSocket > transport/internet/websocket: failed to dial to (ws://IP地址/1469f7da):  > dial tcp IP地址: operation was canceled] > common/retry: all retry attempts failed。
每次都是一个新端口只能用一天，第二天就这样，再建一个新端口，循环往复。
补充：
1、在入站列表看端口均正常开启，检测网站看端口也是open
2、检测网站看 国内和国外都是fail

**Comment by iopq** [comment]
Use a harder to detect technique, like hysteria with automatic port hopping or reality/vision

**Comment by XY-ShadowK** [comment]
被检测封端口了，用tls试试

**Comment by Fubuki0x10DE** [comment]
幾乎唯一可靠的是CN2和TCP。
利申：我已試過無數台non-CN2，KVM和OpenVZ都試過了，TCP到UDP都試過了。

**Comment by xiaorong61** [comment]
Shadowsocks 2022
